### PR TITLE
[PD-4343] feat: implemented storage plans

### PR DESCRIPTION
**Pull request by gabrielmouallem**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ latitudeShApi.Profile.get().then((response) => {
 - `Plans.get`. Params: `(planId, searchParams)`. [Reference](https://docs.latitude.sh/reference/get-plan)
 - `Plans.list`. Params: `(searchParams)`. [Reference](https://docs.latitude.sh/reference/get-plans)
 - `Plans.operatingSystems`. Params: `(searchParams)`. [Reference](https://docs.latitude.sh/reference/get-plans-operating-system)
+- `Plans.storages`. Params: `(searchParams)`. [Reference](https://docs.latitude.sh/reference/get-storage-plans)
```
Comment (marcelo-sousa-mh) [inline README.md:51]: is the plural naming l here necessary? why not `Plans.storage` ?

Comment (gabrielmouallem) [reply README.md:51]: Just did the plural for standardization since `operatingSystems` was on plural.

Comment (esoubihe) [reply README.md:51]: I agree it should be `Plans.storage`

File: lib/resources/plans/index.js [hunk 2]
```diff
@@ -42,6 +42,16 @@ class Plans {
       searchParams
     );
   }
+
+  storages(searchParams = '') {
+    console.log('Hit the storages method');
```
Comment (esoubihe) [inline lib/resources/plans/index.js:47]: Unwanted log?

File: README.md (hunk 1 above)
Comment (gabrielmouallem) [reply README.md]: solved! ty guys!

**Review by marcelo-sousa-mh — APPROVED** [review]

**Pull request merged**